target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -607,7 +608,8 @@ ramips_setup_macs()
 		wan_mac=$(macaddr_add "$lan_mac" 1)
 		;;
 	iodata,wn-ac1167gr|\
-	iodata,wn-ac733gr3)
+	iodata,wn-ac733gr3|\
+	iodata,wnpr2600g)
 		wan_mac=$(macaddr_add "$(mtd_get_mac_binary factory 4)" -1)
```

Comment:
Is this somewhere on flash, too, or is it really necessary to calculate?

Reply:
Replaced to ```wanaddr``` in u-boot-env.